index.html

```diff
@@ -551,6 +551,70 @@ <h2>Exposing MediaStreamTrack source background blur support</h2>
 };</pre>
     </div>
   </section>
+  <section>
+    <h2>Exposing MediaStreamTrack source eye gaze correction support</h2>
+    <p>Some platforms or User Agents may provide built-in support for human eye
+       gaze correction, in particular for camera video streams.
+       This may for instance allow the web application to update its UI or to
+       not apply human eye gaze correction on its own.
+       For that reason, we extend {{MediaStreamTrack}} with the following
+       properties.
+    </p>
+    <p>The WebIDL changes are the following:</p>
+    <pre class="idl"
+>partial dictionary MediaTrackSupportedConstraints {
+  boolean eyeGazeCorrection = true;
+};
+
+partial dictionary MediaTrackCapabilities {
+  sequence&lt;boolean&gt; eyeGazeCorrection;
+};
+
+partial dictionary MediaTrackConstraintSet {
+  ConstrainBoolean eyeGazeCorrection;
+};
+
+partial dictionary MediaTrackSettings {
+  boolean eyeGazeCorrection;
+};</pre>
+    <section>
+      <h3>Processing considerations</h3>
+      <p>When the "eyeGazeCorrection" setting is set to <code>true</code> by
+         the <a>ApplyConstraints algorithm</a>, the User Agent will attempt to correct
+         human eye gaze.
+      </p>
+      <p>When the "eyeGazeCorrection" setting is set to <code>false</code> by
+         the <a>ApplyConstraints algorithm</a>, the User Agent will not correct human
+         eye gaze.
+      </p>
+    </section>
+    <section>
+      <h3>Examples</h3>
+      <pre class="example">
+&lt;video&gt;&lt;/video&gt;
+&lt;script&gt;
+// Open camera.
+const stream = await navigator.mediaDevices.getUserMedia({video: true});
+const [videoTrack] = stream.getVideoTracks();
+
+// Try to correct eye gaze.
+const videoCapabilities = videoTrack.getCapabilities();
+if ((videoCapabilities.eyeGazeCorrection || []).includes(true)) {
+  await videoTrack.applyConstraints({
+    advanced: [{eyeGazeCorrection: true}]
+  });
```

Comment:
Any reason why we use `advanced` constraints? We shouldn't be using those in any of the examples.
```suggestion
  await videoTrack.applyConstraints({eyeGazeCorrection: true});
```